commune_app/all_models/votes.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from .chores import Chore
+from .users import User
+
+
+class Vote(models.Model):
```

Comment:
Missing register of this model in commune_app/admin.py
because of it, we can't see this model in the admin UI in:
http://127.0.0.1:8000/admin/